test/test.js

```diff
@@ -1574,6 +1574,46 @@
       'version': '11.0'
     },
 
+    'Ladybird on Linux x86_64': {
+      'ua': 'Mozilla/5.0 (Linux; x86_64) Ladybird/1.0',
+      'layout': 'LibWeb',
+      'name': 'Ladybird',
+      'os': 'Linux 64-bit',
+      'version': '1.0'
+    },
+
+    'Ladybird on OSX AArch64': {
+      'ua': 'Mozilla/5.0 (MacOS; AArch64) Ladybird/1.0',
+      'layout': 'LibWeb',
+      'name': 'Ladybird',
+      'os': 'MacOS',
```

Comment:
the UA string for macOS uses the official spelling for the OS (`macOS`, not `MacOS`)

Reply:
Thank you AK.